docs/components/tabs.md

```diff
@@ -0,0 +1,292 @@
+---
+sidebarDepth: 0
+---
+# Tabs
+
+**KTabs** - A mindblowing tabs component
+<ClientOnly>
+  <KTabs :tabs="tabs">
+    <template v-slot:tab1>
+      <p>Tab 1 content</p>
+    </template>
+    <template v-slot:tab2>
+      <p>Tab 2 content</p>
+    </template>
+  </KTabs>
+</ClientOnly>
+
+```vue
+<KTabs :tabs="tabs">
+  <template v-slot:tab1>
+    <p>Tab 1 content</p>
+  </template>
+  <template v-slot:tab2>
+    <p>Tab 2 content</p>
+  </template>
+</KTabs>
+
+<script>
+export default {
+  data () {
+    return {
+      tabs: [
+        {
+          hash: '#tab1',
+          title: 'TAB 1'
+        },
+        {
+          hash: '#tab2',
+          title: 'TAB 2'
+        }
+      ]
+    }
+  }
+}
+</script>
+```
+
+## Props
+### tabs
+KTabs takes one required prop which is an array of tab objects.
+
+- `tabs`
+- `@change` - Emitted when a tab is changed
+
+```vue
+<template>
+  <KTabs :tabs="tabs" />
+</template>
+<script>
+export default {
+  data () {
+    return {
+      tabs: [
+        { hash: '#pictures', title: 'Pictures' },
+        { hash: '#movies', title: 'Movies' },
+        { hash: '#books', title: 'Books' },
+      ]
+    }
+  }
+}
+</script>
+```
+
+### default-tab
+By default the tabs will set the first tab in the array as active. You can override this by passing in the hash of any other tab to be used instead.
+<ClientOnly>
+  <KTabs
+    default-tab="#tab2"
+    :tabs="tabs">
+    <template v-slot:tab1>
+      <p>Tab 1 content</p>
+    </template>
+    <template v-slot:tab2>
+      <p>Tab 2 content</p>
+    </template>
+  </KTabs>
+</ClientOnly>
+
+```vue
+<KTabs
+  default-tab-hash="#tab2"
```

Comment:
I had a prop called "initialValue" with vue-autosuggest, only to later realize that users preferred `v-model`. I think we should do the same here. It means it would be reactive as well so you could `v-model="$route.hash"` and wouldn't have to worry about the change event.

Reply:
some prior art:
https://bootstrap-vue.js.org/docs/components/tabs
https://quasar.dev/vue-components/tabs

Reply:
updated to use v-model dawg